flink-core/src/main/java/org/apache/flink/api/common/JobExecutionResult.java

```diff
@@ -112,6 +113,18 @@ public long getNetRuntime(TimeUnit desiredUnit) {
 			.collect(Collectors.toMap(Map.Entry::getKey, entry -> entry.getValue().getUnchecked()));
 	}
 
+	public void printResult() {
```

Comment:
I think we could even have this (or some version of it) as a `toString()`. I don't quite like methods that just print to `stdout`.

Reply:
Yes I've think of this. Let me try to do it.